files/job_wrapper.sh

```diff
@@ -24,25 +24,51 @@ populate(){
     $_CONDOR_SCRATCH_DIR/parrot_static_run -T 30 cp --no-clobber -r "/chirp/CONDOR/$reana_workflow_dir" "$_CONDOR_SCRATCH_DIR/$parent"
 }
 
-# Discover singularity binary path
-# by trying different methods
-find_singularity(){
-    # Method 1: Find in environment
-    singularity_path="$(which singularity 2>/dev/null)"
-    if [ "x$singularity_path" != "x" ]; then return 0; fi
-    
-    # Method 2: Use module, then find in environment
-    alias module 2>/dev/null
-    if [ $? == 0 ];then
-        MODULE_LIST=(singularity tacc-singularity)
-        for var in ${MODULE_LIST[*]}; do
-            module load $var 2>/dev/null
-            singularity_path="$(which singularity 2>/dev/null)"
-            if [ "x$singularity_path" != "x"]; then return 0; fi
-        done
-    fi
+# Discover the container technology available.
+# Currently searching for: Singularity or Shifter.
+# Returns 0: Successful discovery of a container
+#         1: Couldn't find a container
+find_container(){
+    declare -a search_list=("singularity" "shifter")
+    declare -a module_list=(singularity tacc-singularity)
```

Comment:
Just logged into bluewaters and got into a node interactively. I had to type `module load shifter`, so it seems that should also get in the list.

Reply:
You're absolutely right, not sure why I forgot to add that...

Reply:
Resolved in e4991c7495b8e0ca7034144a040d23d03be0a819